_docs/style/text-conventions/writing-guidelines.md

```diff
@@ -0,0 +1,292 @@
+
+## Maintain parallel construction
+
+Parallel construction means that parallel ideas must be presented in parallel grammatical form. This means that each part of a sentence or sentence fragment that serves the same function uses the same grammatical structure.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+When writing a series, it is important that each element in the series performs the same function, that of the object of the verb.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| This stack includes a Cloud Load Balancer, a Cloud Database, and a Master server. | This stack includes a Cloud Load Balancer, a Cloud Database, and Master server. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Parallel structure also applies to options you present to users. Don't ask users to choose from unrelated values.
+
+In this example, **Always** and **Never** are opposites. **Always** and **Blocked** are not opposites.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| <input type="radio" name=""><label for="always"> Always</label> | <input type="radio" name=""><label for="always"> Always</label> |
+| <input type="radio" name=""><label for="stream"> Only with streaming</label> | <input type="radio" name=""><label for="stream"> Only with streaming</label> |
+| <input type="radio" name=""><label for="never"> Never</label> | <input type="radio" name=""><label for="blocked"> Blocked</label> |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+When you document a procedure, ensure that each step of the procedure begins with a verb and contains an object.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| 1. Download form. | 1. Download form. |
```

Comment:
Now that i see it, this structure is still weird.  

put all three "steps" in a single cell.